html2text/__init__.py

```diff
@@ -186,7 +186,9 @@ def handle_charref(self, c):
         self.handle_data(self.charref(c), True)
 
     def handle_entityref(self, c):
-        self.handle_data(self.entityref(c), True)
+        ref = self.entityref(c)
```

Comment:
It would be better to have some comments here, explaining why you have decided to do this.
You may just write the whole comment at https://github.com/Alir3z4/html2text/pull/206#issuecomment-405082665 in here.